modules/sov-ibc/src/rpc/helpers.rs

```diff
@@ -40,3 +43,69 @@ pub fn to_jsonrpsee_error(err: impl ToString) -> ErrorObjectOwned {
         None::<String>,
     )
 }
+
+/// Trait for proof agnostic storage value retrieval.
+pub trait StorageValue<V> {
+    type Output;
+    fn value_at_key<K, C, S>(
+        key: &K,
+        store: &StateMap<K, V, C>,
```

Comment:
Let's rename the `store` to e.g. `state_map` to be more relatable.

Reply:
b017029